src/turbovote/pedestal_toolbox/response.clj

```diff
@@ -15,3 +15,12 @@
   (-> "Unsupported media type"
       ring-resp/response
       (ring-resp/status 415)))
+
+(defn string-bytes [string]
+  (alength (.getBytes string "UTF-8")))
```

Comment:
For *us* this might be a reasonable assumption, but I don't think for a library like this we should be forcing string encoding here.